environments/echis/app-processes.yml

```diff
@@ -11,16 +11,13 @@ celery_processes:
   echis_server0:
     celery,export_download_queue,email_queue,case_import_queue:
       concurrency: 1
-      max_tasks_per_child: 5
```

Comment:
```max_tasks_per_child``` specifies how many tasks can be run until a worker child has to be recycled.

In theory this can prevent memory leaks.  In practice, it's not much of an issue on our systems, especially when a queue is not backlogged since celery powers down worker children that are not in active use.

The motive for this change is to workaround this bug: https://github.com/celery/celery/issues/5120

In the example you commented on, this worker will still only have one child process (determined by ```concurrency```).

Reply:
Cool, thanks